pages/phases/certificate.md

```diff
@@ -20,16 +20,27 @@ roles: [hr, hiring-mgr]
   {% include toolkit/certificate.html %}
 </div>
 
-## HR Preference Adjudication
+<div class="usa-alert usa-alert--info" >
+  <div class="usa-alert__body">
+    <h3 class="usa-alert__heading">Do not conduct a separate minimum qualification review</h3>
+    <p class="usa-alert__text">
+      After SMEs have completed interviews and identified the applicants who are found to meet qualifications, do not conduct a separate minimum qualification review. For more information, see "<a href="{{ site.baseurl }}/about/differences/#sme-assessment">How our process is different</a>."
+    </p>
+  </div>
+</div>
+
+## Adjudicating Preference
+
+Apply Veterans' preference only to applicants who achieve an overall passing score. Up until that point, any applicant can be removed from consideration if they are determined not to qualify based on the required competencies and proficiencies.
 
-Veterans' Preference will be applied only to those who meet the overall passing score. Up until that point, any applicant can be removed from consideration if they are determined not to meet the required qualifications (competencies/proficiencies).
+OPM's MSAC ACE will review the final category assignments to ensure they're lawful. HR will then issue the certificate of eligibles of the "Best Qualified" category to the HM.
 
-## Hiring manager interviews
+## Post-Certificate Interviews
 
-The primary variable changed by this process is how you assess applicants to determine they meet the basic qualifications before applying rankings and preference. All steps after the certificate can be done the same as the agency currently does. If hiring managers want to have additional interviews with applicants from the certificate they can do so. However, to pass over a veteran, they need to abide by title 5 requirements and submit a pass over request.
+After the hiring certificate is issued, agencies can use the same procedures they always have. Hiring managers may conduct additional interviews with applicants from the certificate, or choose instead to rely on the interview transcript notes to determine whether they'll make offers to applicants on the certificate.
 
-Hiring managers may also choose not to conduct additional interviews and instead to rely on the interview transcript notes to determine if they want to make offers to applicants on the certificate.
+To pass over a veteran, hiring managers must meet the requirements of 5 U.S.C. § 3318 and submit a pass over request. For more information, see the "[Vet Guide for HR Professionals](https://www.opm.gov/policy-data-oversight/veterans-services/vet-guide-for-hr-professionals/)" on the *OPM Website*.
```

Comment:
add- it's unlikely they will be able to pass over a vet for lack of qualifications. however, they could submit a passover request if the applicant isnt a good fit for the office as deemed from reference checks and/or HM interviews